spec/fixtures/files/watched/user@domain.com/2023_January.json

```diff
@@ -0,0 +1,57 @@
+{
+  "type": "FeatureCollection",
```

Comment:
This file is also not representing real `YEAR_Month.json` file structure from google takeout. Example: https://github.com/Freika/dawarich/blob/master/spec/fixtures/files/google/semantic_history.json#L0-L1